app/views/pages/accessibility.md

```diff
@@ -2,61 +2,90 @@
+## Reporting accessibility problems with Register trainee teachers
 
-<h2 class='govuk-heading-m'>Non-accessible content</h2>
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact: <a class='govuk-link' href='mailto:becomingateacher@digital.education.gov.uk?subject=Accessibility%20issues%20'>becomingateacher@digital.education.gov.uk</a>. Include ‘Accessibility’ in the subject line of your email.
 
-<p class='govuk-body'>The content listed below is non-accessible for the following reasons.</p>
+## Enforcement procedure
+The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, <a href='https://www.equalityadvisoryservice.com/' class='govuk-link' rel='noreferrer noopener' target="_blank">contact the Equality Advisory and Support Service (opens in a new tab)</a>.
 
-<h3 class='govuk-heading-s'>Non-compliance with the accessibility regulations</h3>
+## Technical information about this service’s accessibility
 
-<p class='govuk-body'>To disable auto-fill on text inputs, we need to set the autocomplete attribute to a non-standard value. This fails <a href='https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html' class='govuk-link'> WCAG 1.3.5: Identify Input Purpose</a>.</p>
+The Department for Education is committed to making its websites accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.
 
-<p class='govuk-body'>Auto-fill is not suitable for Register trainee teachers and makes the task of registering trainee teachers more difficult.</p>
+## Compliance status
 
-<p class='govuk-body'>We believe the impact of a non-standard value is minimal and the value of disabling auto-fill is much greater. The fail is unlikely to impact our users and will benefit many of them.</p>
+This website is partially compliant with the WCAG (Web Content Accessibility Guidelines) version 2.2 AA standard, due to the following non-compliances (and/or) the following exemptions.
 
-<p class='govuk-body'><a href='https://bat-design-history.netlify.app/register-trainee-teachers/preventing-browser-auto-fill/' class='govuk-link'>Read more about why and how we’ve prevented auto-fill in Register</a></p>
+## Non-accessible content
 
+The content is non-accessible for the following reasons:
 
-<h2 class='govuk-heading-m'>Preparation of this accessibility statement</h2>
+### Specific error suggestions are not provided for some fields on forms
+This can be seen on date fields, if the user inputs an incorrectly formatted date, the error message does not provide a suggestion on how to rectify it. This fails WCAG 2.2 success criterion 3.3.1 Error Identification.
```

Comment:
```suggestion

This can be seen on date fields, if the user inputs an incorrectly formatted date, the error message does not provide a suggestion on how to rectify it. This fails WCAG 2.2 success criterion 3.3.1 Error Identification.
```